File: Examples/Ruby/generating-saving/cloud-storage/generate_a_codablockf_type_barcode.rb
#ExStart:
require 'aspose_barcode_cloud'

class GenerateBarcode

  include AsposeBarcodeCloud
  include AsposeStorageCloud

  def initialize
    #Get App key and App SID from https://cloud.aspose.com
    AsposeApp.app_key_and_sid("", "")
    @barcode_api = BarcodeApi.new
  end

  def upload_file(file_name)
    @storage_api = StorageApi.new 
    response = @storage_api.put_create(file_name, File.open("../../../data/" << file_name,"r") { |io| io.read } )
  end

  # Set/create document property.
  def generate_a_codablockf_type_barcode
      opts = {text: "(01)23456789012", type: "codablockF", format: "jpg"}
      response = @barcode_api.get_barcode_generate(opts)
  end

end

generateBarcode = GenerateBarcode.new()
puts generateBarcode.generate_a_codablockf_type_barcode
#ExEnd:

File: Examples/PHP/generating-saving/cloud-storage/generate-a-codablockf-type-barcode.php
//ExStart:
<?php
require_once realpath(__DIR__ . '/../..') . '/vendor/autoload.php';
require_once realpath(__DIR__ . '/../..') . '/Utils.php';

use Aspose\Barcode\BarcodeApi;
use Aspose\Barcode\AsposeApp;

class GenerateBarcode {

	public $barcodeApi;

	public function __construct() {
		AsposeApp::$appSID = Utils::appSID;
		AsposeApp::$apiKey = Utils::apiKey;
		$this->barcodeApi = new BarcodeApi();
	}

	public function generateACodablockFTypeBarcode() {

		$text = "(01)23456789012";
        $type = "codablockF";
        $format = "jpg";
           
        $result = $this->barcode->GetBarcodeGenerate($text, $type, $format, $resolutionX = null, $resolutionY = null, $dimensionX = null, $dimensionY = null, $enableChecksum = null);
        $fh = fopen(realpath(__DIR__ . '/../../../..') . '/Data/Output/barcode_out.jpg', 'w');
        fwrite($fh, $result);
        fclose($fh);
	}
}

$generateBarcode = new GenerateBarcode();
$generateBarcode->generateACodablockFTypeBarcode();

?>
//ExEnd: